script/node-disabled-tests.json

```diff
@@ -273,6 +277,9 @@
   "message/undefined_reference_in_new_context",
   "message/unhandled_promise_trace_warnings",
   "message/util_inspect_error",
+  "message/source_map_throw_catch",
```

Comment:
we don't support source maps for the same reason we don't seem to support coverage and i'd rather investigate why outside this PR than block it on that since source maps are new and experimental anyways